platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -116,11 +122,26 @@ public void doExport(ExportMigrationContext context) {
   public void doImport(ImportMigrationContext context) {
     final ImportMigrationConfigurationAdminContext adminContext =
         new ImportMigrationConfigurationAdminContext(
-            context, this, configurationAdmin, getConfigurations(context));
-
+            context, this, configurationAdmin, getConfigurations(context), true);
     adminContext.memoryEntries().forEach(ImportMigrationConfigurationAdminEntry::restore);
   }
 
+  @Override
+  public void doVersionUpgradeImport(ImportMigrationContext context) {
+    final ImportMigrationConfigurationAdminContext adminContext =
+        new ImportMigrationConfigurationAdminContext(
+            context, this, configurationAdmin, getConfigurations(context), false);
+
+    adminContext
+        .memoryEntries()
+        .filter(
```

Comment:
✏️ Time crate a private method that receives an entry and returns true or false where you put that filter. This will make the stream more readable